spyder/utils/qthelpers.py

```diff
@@ -311,7 +311,10 @@ def add_actions(target, actions, insert_before=None):
                     except RuntimeError:
                         continue
             if insert_before is None:
-                target.addAction(action)
+                try:
+                    target.addAction(action)

```

Comment:
@dalthviz some context of why was this happening or a reference to the issue, is good for bookkeeping :-)